### ISSUE TITLE
Countdown doesn't seem to be working? 

### ISSUE THREAD

**Issue by mmk8283** [ronbal/photobooth#12]
Trying to get this up and running and it doesn't seem to be doing the countdown

**Comment by ronbal** [comment]
Any error messages?  did you run the setup.py on a new installation?

**Comment by mmk8283** [comment]
I did two fresh setups. no errors. seems the code is calling for (test) in the countdown area... there is a test.txt but it is a blank file

**Comment by ronbal** [comment]
Which distribution? I've tested with jessy and stretch

**Comment by mmk8283** [comment]
Stretch.

**Comment by ronbal** [comment]
You should uncomment lin 124 for window mode. The post the message from debug window.

**Comment by mmk8283** [comment]
I will give that a try, Thanks

**Comment by mmk8283** [comment]
If I define a window size on line #123 like the debug line has, it works just fine. I'm running this on a larger monitor. If I try to change the preview to anything larger than 640x480 the countdown stops working again.

**Comment by ronbal** [comment]
So you see errors  now?

**Comment by mmk8283** [comment]
When I uncommented line 124 the countdown started working as it should. I commented out again, and problem came back. so I added the window definition from line 124 to line 123 and everything works fine. I did notice any debug errors while I had line 124 uncommented. 

It seems the problem is solved as long as the window is defined in line 123. however, I would like to increase the size of it. any attempts I made result in the problem showing up all over again. 
 
Thanks for your help btw. :)
